src/harness/testcases/WINNF_FT_S_RLQ_testcase.py

```diff
@@ -212,6 +212,185 @@ def test_WINNF_FT_S_RLQ_2(self):
+    self.assertIn(response[0]['response']['responseCode'], 103)
+    self.assertEqual(response[1]['cbsdId'], cbsd_id_2)
+    self.assertFalse('grantId' in response[1])
+    self.assertEqual(response[1]['response']['responseCode'], 103)
+    self.assertFalse('cbsdId' in response[2])
+    self.assertFalse('grantId' in response[2])
+    self.assertIn(response[2]['response']['responseCode'], 103)
+    self.assertEqual(response[3]['cbsdId'], cbsd_id_2)
+    self.assertFalse('grantId' in response[3])
+    self.assertEqual(response[3]['response']['responseCode'], 103)
+    self.assertEqual(response[4]['cbsdId'], cbsd_id_2)
+    self.assertFalse('grantId' in response[4])
+    self.assertEqual(response[4]['response']['responseCode'], 103)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_RLQ_4(self):
+    """CBSD relinquishment request of multiple grants
+
+    CBSD Harness sends Relinquishment Request to SAS including multiple
+    grants with valid CBSD ID and valid Grant ID, but with protocol
+    version not supported by SAS. The response should be FAIL.
+    """
+
+    # Register the device
+    device = json.load(open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    self._sas_admin.InjectFccId({'fccId': device['fccId']})
+    self._sas_admin.InjectUserId({'userId': device['userId']})
+    request = {'registrationRequest': [device]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
+
+    # Check registration response
+    self.assertEqual(response['response']['responseCode'], 0)
+    cbsd_id = response['cbsdId']
+    del request, response
+
+    # Request grants
+    grant_1 = json.load(open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_1['cbsdId'] = cbsd_id
+    grant_1['operationParam']['operationFrequencyRange'] = {
+         'lowFrequency': 3600000000,
+         'highFrequency': 3610000000
+    }
+    grant_2 = json.load(open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_2['cbsdId'] = cbsd_id
+    grant_2['operationParam']['operationFrequencyRange'] = {
+         'lowFrequency': 3610000000,
+         'highFrequency': 3620000000
+    }
+    grant_3 = json.load(open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_3['cbsdId'] = cbsd_id
+    grant_3['operationParam']['operationFrequencyRange'] = {
+         'lowFrequency': 3620000000,
+         'highFrequency': 3630000000
+    }
+    grant_id = []
+    request = {'grantRequest': [grant_1, grant_2, grant_3]}
+    response = self._sas.Grant(request)['grantResponse']
+
+    # Check grant response
+    self.assertEqual(len(response), 3)
+    for resp in response:
+      self.assertEqual(resp['response']['responseCode'], 0)
+      self.assertEqual(resp['cbsdId'], cbsd_id)
+      grant_id.append(resp['grantId'])
+    del request, response
+
+    # Save sas version
+    version = self._sas._sas_version
+    # Use higher than supported version
+    self._sas._sas_version = 'v1.3'
```

Comment:
"v1.3" might not be a safe value to use since we are already at v1.2 and could end up with v1.3 before everything is final. v3.0 would be safer.